things/service_test.go

```diff
@@ -557,6 +582,20 @@ func TestCreateChannels(t *testing.T) {
 			token:    wrongValue,
 			err:      things.ErrUnauthorizedAccess,
 		},
+		{
+			desc:   "create new channels with external UUID",
+			channels: []things.Channel{{ID: "fe6b4e92-cc98-425e-b0aa-000000000001", Name: "a", Metadata: meta},
```

Comment:
Same here, create a `var` for these lists

Reply:
Used var here too for channel